lib/plugins/logMetrics.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var bunyan = require('bunyan');
+
+function logMetrics(options) {
```

Comment:
Classes should start with a capital letter. Let's call this MetricBuffer instead of logMetrics.